content/en/xtc/loadtesting/155-lt-settings.md

```diff
@@ -35,27 +35,25 @@ Overwriting the Load Profile with a Load Function
 
 Learn more in our _Load Testing_ section about [configuring load profiles]({{< relref "/xlt/load-testing/manual/470-load-configuration" >}}) for your test run.
 
-### Machine Configuration
+### Properties
+In _Properties_ and _Secret Properties_ you may overwrite [properties]({{< relref "/xlt/load-testing/manual/480-test-suite-configuration" >}}) for test execution. If nothing is set here, the properties from the project repository will be used. Properties may also be overwritten globally for all load tests of this project in the [project configuration]({{< relref "120-load-project-configuration#properties" >}}).
```

Comment:
The properties from the repo are always used, XTC properties are always an overwrite. Please verify that.

Reply:
Hope it is more clear now?